papers/whitepaper.md

```diff
@@ -0,0 +1,93 @@
+
+The development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will be crucial to democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work, allowing for experts in the community to earn a continuous stream of revenue through their model utilization.
+
+This is especially important once we factor in possible risks of AI censorship. For example, Italy [banned](https://www.bbc.com/news/technology-65139406) the use of OpenAI's ChatGPT shortly after its release, due to user privacy concerns (even though it later lifted the ban). On the other hand, companies like OpenAI are self-censoring due to societal and political reasons. For example, OpenAI will not predict the next presidential election, etc. These use cases will likely create large predictions/content markets that big technology companies will not want to be involved in. The Atoma Network, on the other hand, can leverage open-source software AI models, at a lower-cost GPU cluster to perform inference for these types of use cases.
+
+The Atoma Network will be powered by its TOMA token. Through robust tokenomics, compute providers, protocol participants, and developers will be rewarded by their contribution to maintaining the network longevity. The TOMA token will be required to pay for AI inference fees, for nodes to deposit collateral to be able to participate on the network, and also for developers to be paid for their contributions to the Atoma Network.
+
+The token distribution will follow an initial period of inflation in order to subsidize the network's growth. This growth will be used to subsidize future node operators to acquire powerful AI hardware, to incentivize model creators to develop new AI models, specialized for applications of interest to the Atoma Network (such as financial market predictions, etc), as well as to incentivize developers to build applications that drive demand for the available compute resources available on the Atoma Network. After the network's inflationary period (which will last a few years), the token emission distribution will follow a slightly deflationary dynamics, through a combination of tail emissions and burning mechanisms. Together, both strategies will lead to a gradual decrease in the total supply of the TOMA token, contributing for its overall value increase (provided the demand for Atoma Network increases), while also allowing for the allocation of the very small portion of newly minted tokens to the protocol's treasury that can then be used for different purposes, which can include bringing new generation hardware to the network, social incentives through UBI, etc.
```

Comment:
The token distribution will follow an initial period of inflation in order to subsidize the network's growth. This growth will be used to:
1. subsidize future node operators to acquire powerful AI hardware, 
2. incentivize model creators to develop new AI models, specialized for applications of interest to the Atoma Network (such as financial market predictions)
3. incentivize developers to build applications that drive demand for the available compute resources available on the Atoma Network. 

After the network's inflationary period (which will last a few years), the token emission distribution will follow a slightly deflationary pattern through a combination of tail emissions and burning mechanisms. Together, both strategies will lead to a gradual decrease in the total supply of the TOMA token, contributing for its overall value increase (provided the demand for Atoma Network increases), while also allowing for the allocation of the very small portion of newly minted tokens to the protocol's treasury. These newly minted tokens can then be used for different purposes including bringing new generation hardware to the network or social incentives through UBI.